server/db/models/Gig.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function (sequelize, DataTypes) {
+  var Gig = sequelize.define('Gig', {
+    title: DataTypes.STRING,
+    type: DataTypes.ENUM('wedding', 'birthday', 'business'),
```

Comment:
We should chat about Gig.type once we get some of the real data. I can see us running into problems trying to scale this and also running into problems when our app gets used by different catering companies